packages/cli/src/commands/prerender.js

```diff
@@ -158,48 +184,32 @@ const diagnosticCheck = () => {
   }
 }
 
-export const handler = async ({
-  path: routerPath,
-  output,
-  dryRun,
-  verbose,
-}) => {
-  if (routerPath) {
-    await runPrerender({
-      routerPath,
-      outputHtmlPath: output,
-      dryRun,
-    })
-
-    return
-  }
-
-  const listrTasks = getTasks(dryRun)
+export const handler = async ({ path: routerPath, dryRun, verbose }) => {
+  const listrTasks = await getTasks(dryRun, routerPath)
 
   const tasks = new Listr(listrTasks, {
     renderer: verbose ? VerboseRenderer : 'default',
-    concurrent: true,
   })
 
   try {
+    if (dryRun) {
+      console.log(c.info('::: Dry run, not writing changes :::'))
+    }
+
     await tasks.run()
   } catch (e) {
     console.log()
     await diagnosticCheck()
 
-    console.log(
-      c.warning(
-        'Not all routes were succesfully prerendered. Run `yarn rw prerender --dry-run` for detailed logs'
-      )
-    )
+    console.log(c.warning('Tips:'))
     console.log(
       c.info(
-        `We could not prerender all your pages, but your Redwood app should still work fine.`
+        `- This could mean that a library you're using does not support SSR.`
       )
     )
     console.log(
       c.info(
-        `This could mean that a library you're using does not support SSR.`
+        '- Avoid using `window` in your render functions without checks. \n  See https://redwoodjs.com/docs/prerender#prerender-utils'
```

Comment:
```suggestion
        '- Avoid using `window` in your React components without checks. \n  See https://redwoodjs.com/docs/prerender#prerender-utils'
```
Not sure your average developer would know what a "render function" is now that everyone is always just using FCs.

Reply:
Thing is, its totally fine to use it in callbacks/effects - just not in the main render function block

Reply:
Yeah of course because they're not executed on the first pass through the code. How about this?

```suggestion
        '- Avoid using `window` in the initial render path through your React components without checks. \n  See https://redwoodjs.com/docs/prerender#prerender-utils'
```